test/lib/authentication.spec.ts

```diff
@@ -1,7 +1,7 @@
 import chai from 'chai';
 import chaiExclude from 'chai-exclude';
 import { mockChtSession } from '../mocks';
-import Auth from '../../dist/lib/authentication';
+import Auth from '../../dist/src/lib/authentication';
```

Comment:
I think this was a bug. I don't think this should be loading from the `dist` folder.